enzy_htp/structure/metal_atom.py

```diff
@@ -101,13 +115,16 @@ def get_donor_atoms(self, method: str = "ionic", check_radius=4.0) -> List[Atom]
         # find radius for matal
         r_metal = self.atom.radius(method)
         # get target with in check_radius (default: 4A)
-        range_atoms = self.chain.parent.find_atoms_in_range(self.coord, range_distance=check_radius)
+        range_atoms = self.chain.parent.find_atoms_in_range(
+            self.coord, range_distance=check_radius)
         atom: Atom
         for atom in range_atoms:
             # only check donor atom (by atom name)
             if atom.is_donor_atom():
                 r_donor = atom.radius(method)
-                if atom.distance_to(self.atom) - (r_metal + r_donor) <= 0.01: # TODO refine this approximation
+                if atom.distance_to(self.atom) - (
+                        r_metal +
+                        r_donor) <= 0.01:  # TODO refine this approximation
```

Comment:
Maybe change yapf's behavior on line-too-long a bit? (Exclude characters come from comments? increase the allowed line length a bit?) Because lines like this looks harder too read.

Reply:
Sure we can look into this. This is the default behavior when google style is enabled

Reply:
Yeah I wonder if any configurations can use for it

Reply:
Updated and fixed